src/EnergyPlus/CurveManager.cc

```diff
@@ -6379,6 +6379,42 @@ Label999: ;
 		return CurveOrTableObjectTypeNum;
 	}
 
+	void
+	checkCurveIsNormalizedToOne( 
+		std::string const callingRoutineObj, // calling routine with object type
+		std::string const objectName,        // parent object where curve is used
+		int const curveIndex,                // index to curve object
+		std::string const cFieldName,        // object field name
+		std::string const cFieldValue,       // user input curve name
```

Comment:
This argument isn't really needed, since the curve name is known from the index, but probably not worth changing all of these now.  Otherwise, much better with this new function.